extensions-api/src/main/scala/pl/touk/nussknacker/engine/deployment/DeploymentId.scala

```diff
@@ -3,15 +3,21 @@ package pl.touk.nussknacker.engine.deployment
 import io.circe.generic.extras.semiauto.{deriveUnwrappedDecoder, deriveUnwrappedEncoder}
 import io.circe.{Decoder, Encoder}
 import pl.touk.nussknacker.engine.api.deployment.ProcessActionId
+import pl.touk.nussknacker.engine.newdeployment
 
 import java.util.UUID
 import scala.util.Try
 
 //id generated by Nussknacker - in contrary to ExternalDeploymentId which is generated by e.g. Flink
 // TODO: This class will be replaced by DeploymentId from newdeployment package, see docs in newdeployment.DeploymentService
 final case class DeploymentId(value: String) extends AnyVal {
```

Comment:
Below I see w try to convert string to UUID. Why don't make a value just an UUID type?

Reply:
I already introduced a `newdeployment.DeploymentId` with `UUID` but it currently can't be used everwhere because periodic mechanism uses `PeriodicProcessDeploymentId` which has `Long`.